schemas/didDocument/didDocument.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$id": "/did-core.didDocument",
+    "type": "object",
+    "properties": {
+      "@context": {
+        "title": "@context",
+        "description": "https://w3c.github.io/did-core-registry/#context",
+        "anyOf": [
+          { "type": "string" },
```

Comment:
```suggestion
```

Reply:
This would break compatibility with existing implementations that use a single string.

Reply:
Yes it would. One complaint levied by the JSON-only folks was that anything that could be an array should always be an array so that they can simplify the code written. This was back when `@context` was still allowed in JSON-only documents. It's forbidden now, so we could do whatever we want here that the JSON-LD community would be happy with... BUT, it might harm some of the organizations that want to do limited JSON-only processing (even if they do JSON-LD processing later in their pipeline). 

Digital Bazaar is one such company - we do JSON-only processing in earlier stages of our pipeline before we do full blown JSON-LD processing.

I'm on the fence about this one. @dlongley @davidlehn any strong feelings about whether or not we should allow pure strings as well as arrays of strings to be associated w/ the `@context` value?

Reply:
I think we should allow both until someone argues strongly and convincingly that we should only support an array.

Reply:
I think @awoie had a strong opinion about "always array"... pulling him into this discussion.

Reply:
I agree, especially if `@context` is forbidden in "pure json" representations.

Reply:
> I agree, especially if @context is forbidden in "pure json" representations.

See [above](https://github.com/w3c/did-core-registry/pull/4#discussion_r384745153).

Reply:
Ok, let's keep this for now... note that I suggested a code change that makes the rule more strict (see change suggestion above).

Reply:
yes, thanks for the stricter pattern, we should really be using those everywhere and have some tests around them.